soccer/src/soccer/strategy/agent/position/offense.cpp

```diff
@@ -172,6 +175,10 @@ std::optional<RobotIntent> Offense::state_to_task(RobotIntent intent) {
             planning::MotionCommand{"path_target", current_location_instant, face_ball};
         intent.motion_command = face_ball_cmd;
         return intent;
+    } else if (current_state_ == AWAITING_SEND_PASS) {
```

Comment:
should we really be standing perfectly still and doing nothing while waiting to pass? it's okay for now but let's have a discussion about this and how this entire state machine operates.